packages/editor/src/store/reducer.js

```diff
@@ -885,6 +885,15 @@ export function notices( state = [], action ) {
 	return state;
 }
 
+export function postLock( state = { isLocked: false }, action ) {
```

Comment:
I was going to recommend that the shape here is more complex than it needs to be, since we only track a boolean, why not just have the reducer as `isPostLocked() : boolean`. But then I realized we're tracking more than just a boolean in this object. My thought-process here can serve as evidence we need to be clearer what specifically we're tracking. This can be helped with JSDoc.